src/Runner.Listener/Configuration/ServiceControlManager.cs

```diff
@@ -68,20 +66,20 @@ public void CalculateServiceName(RunnerSettings settings, string serviceNamePatt
                 // Lets add a suffix with a random number to reduce the chance of collisions between runner names once we truncate
                 var random = new Random();
                 var num = random.Next(1000, 9999).ToString();
-                runnerNameSubstring +=$"-{num}";
+                runnerNameSubstring += $"-{num}";
                 serviceName = StringUtil.Format(serviceNamePattern, repoOrOrgNameSubstring, runnerNameSubstring);
             }
 
             serviceDisplayName = StringUtil.Format(serviceDisplayNamePattern, repoOrOrgName, settings.AgentName);
 
             Trace.Info($"Service name '{serviceName}' display name '{serviceDisplayName}' will be used for service configuration.");
         }
-        #if (OS_LINUX || OS_OSX)
-            const int MaxServiceNameLength = 150;
-            const int MaxRepoOrgCharacters = 70;
-        #elif OS_WINDOWS
+#if (OS_LINUX || OS_OSX)
+        const int MaxServiceNameLength = 150;
+        const int MaxRepoOrgCharacters = 70;
+#elif OS_WINDOWS
             const int MaxServiceNameLength = 80;
```

Comment:
Should the spacing line up with the `const` above?

Reply:
done :)